### DISCUSSION TITLE
v1.17.0

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#643] [General]
## Features
- Added button to show/hide charts on the dashboard
- Default theme now uses user's system preference
- `IGNORE_IP` environment variable now accepts CIDR notation

## Fixes
- Fixed issue with checkbox requiring two clicks
- Fixed mobile menu display
- Fix page filter issue with trailing slashes

## Updates
- Language updates
- Updated to latest Next.js and Prisma

A huge thanks to all the contributors on this release! 🙏 @ym-project @alex-connor @kiprasmel @Sammy-T @MoltenCoffee @Wonderfall 

<hr /><em>This discussion was created from the release <a href='https://github.com/mikecao/umami/releases/tag/v1.17.0'>v1.17.0</a>.</em>

**Comment by Chaphasilor** [comment]
Auto-Dark mode, finally! :D

**Comment by ayewo** [comment]
@mikecao 

Congrats on the release 💪🏆!

Sorry if this isn’t the right place but how are database schema changes between versions handled by Umami?

I was able to successfully upgrade from v1.4.0 to v1.17 but I did not notice any updates to the database schema in MySQL. 

Does this mean that there were no (minor) schema changes from v1.4.0 through to v1.17.0?

**Comment by mikecao** [comment]
@ayewo Right. There haven't been any schema changes since the initial release. Everything that has changed should be covered in the release notes.
